pkg/utils/logger.go

```diff
@@ -129,3 +130,10 @@ func SetOutFile(name string) {
 		logger.tty = false
 	}
 }
+
+func SetOutput(w io.Writer, istty bool) {
+	for _, logger := range loggers {
+		logger.SetOutput(w)
+		logger.tty = istty
```

Comment:
tty is used by DisableLogColor(), we need to add `colorful` for that rather than using `tty=false`